### PR TITLE
[Proposal]  : Multiple Owner Feature architecture.

### PR DESCRIPTION
**Pull request by aryan-bhokare**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by namkyu1999 — COMMENTED** [review]
can you move the image to `proposals/images`?

**Comment by namkyu1999** [comment]
And is there any db schema changes?

**Comment by aryan-bhokare** [comment]
> And is there any db schema changes?

yes there's only change in the structure of owner.

**Comment by smitthakkar96** [comment]
Thanks, @aryan-bhokare. Is there a way to extend your proposal to allow only certain users to create projects? I am also wondering why we need to create a project on admin login? Can we make project creation an explicit action?

**Comment by aryan-bhokare** [comment]
> Thanks, @aryan-bhokare. Is there a way to extend your proposal to allow only certain users to create projects? I am also wondering why we need to create a project on admin login? Can we make project creation an explicit action?

Surely we will have to discuss about that. But we will be giving and support for deleting the default project which is created on first login.

File: proposals/multipleowner-project.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- Users can create multiple projects.
+
+## Proposal
+
+### Flow Diagram
+![Flow](multipleprojectwoner.png)
+
+### Implementation Details
+
+The proposal suggests decoupling user login and project creation. We are going to support multiple owners and add APIs `Delete Project`: to delete the projects, `Update Role`: to update role of the user in the project. We will also support multiple projects under one user.
```
Comment (Saranya-jena) [inline proposals/multipleowner-project.md:23]: We can also add a couple of more sections: DB schema changes and API changes

Comment (aryan-bhokare) [reply proposals/multipleowner-project.md:23]: sure

**Review by SarthakJain26 — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**